.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "extends": ["standard", "plugin:prettier/recommended"],
+  "env": {
+    "browser": false,
+    "commonjs": true,
+    "es2021": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 12
+  },
+  "rules": {
```

Comment:
please enforce use strict with an eslint rule. that rule is tricky to set up, ask Nigel if you need help

Reply:
I'm not exactly sure what warnings you want from eslint and why

I can set ` "strict": [2, "global"],` which then errors when it finds `'use strict'` with

>  'use strict' is unnecessary inside of modules 

But I think you're asking for the opposite?

Reply:
yep, the opposite, hence why I'm mentioning that it's tricky to set up. see how it's done here https://github.com/nearform/mercurius-apollo-registry